src/nupic/algorithms/KNNClassifier.py

```diff
@@ -95,19 +95,22 @@ class labels
         the p value of the Lp-norm
 
     @param distanceMethod (string) The method used to compute distance between
-        patterns. The possible options are:
+        input patterns and prototype patterns. The possible options are:
         "norm": When distanceNorm is 2, this is the euclidean distance,
                 When distanceNorm is 1, this is the manhattan distance
                 In general: sum(abs(x-proto) ^ distanceNorm) ^ (1/distanceNorm)
         "rawOverlap": Only appropriate when inputs are binary. This computes:
                 (width of the input) - (# bits of overlap between input
                 and prototype).
-        "pctOverlapOfLarger": Only appropriate for binary inputs. This computes
+        "pctOverlapOfInput": Only appropriate for binary inputs. This computes
                 1.0 - (# bits overlap between input and prototype) /
-                        max(# bits in input, # bits in prototype)
+                        (# ON bits in input)
```

Comment:
No because you're first dividing by the width of input, giving a value between 0.0 (no overlapping bits) and 1.0 (all bits of the input overlap). We want to flip this, where the minimum distance is maximal overlap, so we subtract from 1.0.